src/main/io/usb_cdc_hid.c

```diff
@@ -48,25 +49,44 @@
 const uint8_t hidChannelMapping[] = {
     ROLL,     // X
     PITCH,    // Y
-    AUX3,
+    AUX3,     // Z
     YAW,      // X Rotation
     AUX1,     // Z Rotation
     THROTTLE, // Y Rotation
-    AUX4,
-    AUX2,     // Wheel
+    AUX4,     // Slider
+    AUX2,     // Dial
+    AUX5,     // Button 1
+    AUX6,     // Button 2
+    AUX7,     // Button 3
+    AUX8,     // Button 4
+    AUX9,     // Button 5
+    AUX10,    // Button 6
+    AUX11,    // Button 7
+    AUX12     // Button 8
 };
 
 void sendRcDataToHid(void)
 {
-    int8_t report[8];
+    int8_t report[9];
+     // Axes
     for (unsigned i = 0; i < USB_CDC_HID_NUM_AXES; i++) {
         const uint8_t channel = hidChannelMapping[i];
         report[i] = scaleRange(constrain(rcData[channel], PWM_RANGE_MIN, PWM_RANGE_MAX), PWM_RANGE_MIN, PWM_RANGE_MAX, USB_CDC_HID_RANGE_MIN, USB_CDC_HID_RANGE_MAX);
-        if (i == 1) {
-            // For some reason ROLL is inverted in Windows
+        if (channel == PITCH) {
+            // PITCH is inverted in Windows
             report[i] = -report[i];
         }
```

Comment:
Good find.

Reply:
Thanks!
But still... iterator should be in sync with the HID Report definition of the 'usage'. 
Anyway, this is most readable.

And... all this thing with HID drivers is one of the most messy things in BF. LOL.